src/main/java/ladder/domain/Point.java

```diff
@@ -0,0 +1,30 @@
+package ladder.domain;
+
+public enum Point {
+
+  CONNECT(true, "-----|"),
+  DISCONNECT(false, "     |");
+
+  private final boolean isConnected;
+  private final String pedal;
+
+  Point(boolean isConnected, String pedal) {
+    this.isConnected = isConnected;
+    this.pedal = pedal;
+  }
+
+  public static Point getPoint(boolean isConnected) {
+    if (isConnected) {
+      return CONNECT;
+    }
+    return DISCONNECT;
```

Comment:
```suggestion
  public static Point getPoint(boolean isConnected) {
    return isConnected ? CONNECT : DISCONNECT;
```

저는 getPoint 와 같은 static 팩토리 메서드를 허용하려고 합니다. (사실 저는 이런 static 메서드를 좋아하는 엘레강트 하지 않은 개발자랍니다.)
함수형으로 프로그래밍을 할 때 간단한 조건문은 삼항 연산자를 활용해보는 것은 어떨까요?
이것도 개인 취향이 강하게 반영된 것이므로 꼭 따를 필요는 없습니다.

다만, 제가 함수형 프로그래밍에서 삼항 연산자를 좋아하는 이유는 언급하면 좋을 것 같아서,
좀 지나치다 싶은 피드백을 달아봅니다.

한번 expression 과 statement 의 차이에 대해서 공부해보세요.
삼항 연산자는 expression 일까요? statement 일까요?
반대로 if 문은 expression 일까요? statement 일까요?
그리고 함수형 프로그래밍과 절차지향 프로그래밍에서 expression 과 statement 를 어떻게 다룰까요? 혹시 다르게 다루는 차이점이 있을까요?

Reply:
코멘트 주신 내용 찾아보도록 하겠습니다!